tests/tf-manifests/aws/vpc/main.tf

```diff
@@ -11,6 +11,7 @@ terraform {
 
 provider "aws" {
   region = var.aws_region
+  shared_credentials_files = var.aws_shared_credentials_file == null ? null : [var.aws_shared_credentials_file]
```

Comment:
why translating into an array and not having the variable directly into the correct format ?

Reply:
As the value is passed via the system environment starting from Prow framework, we must do a _translation_ somewhere, e.g., in the function `PrepareSharedVpcPolicyAndHostedZone`,  so I think passing the simple string value all the way and converting it inside the VPC module is more straightforward.

Reply:
I would prefer to have the correct format as a variable. and it is the responsibility of the test code to parse the env and provide the correct format to TF

Reply:
Updated.